bids-validator/utils/type.js

```diff
@@ -390,10 +394,10 @@ export default {
   },
 
   checkType(obj, typeString) {
-    if (typeString == 'number') {
+    if (typeString === 'number') {
```

Comment:
Is this change related to Microscopy specifically?
If not, I would recommend not making changes to the already existing code of the validator, to not impact other modalities' validation.